crates/holochain/src/sweettest/sweet_conductor.rs

```diff
@@ -990,6 +991,48 @@ impl SweetConductor {
         })
     }
 
+    /// Get all invalid integrated ops from the DHT database.
+    pub async fn get_invalid_integrated_ops(
+        &self,
+        dht_db: &DbWrite<DbKindDht>,
+    ) -> ConductorApiResult<Vec<DhtOpHashed>> {
+        use holo_hash::DhtOpHash;
+        use holochain_state::query::map_sql_dht_op;
+        use holochain_zome_types::prelude::ValidationStatus;
+
+        let result = dht_db
+            .read_async(move |txn| -> DatabaseResult<_> {
+                let mut stmt = txn.prepare(
+                    "
+                    SELECT
+                    DhtOp.hash as hash, DhtOp.type as dht_type, DhtOp.validation_status,
+                    Action.blob as action_blob, Entry.blob as entry_blob
+                    FROM DhtOp
+                    JOIN Action ON Action.hash = DhtOp.action_hash
+                    LEFT JOIN Entry ON Entry.hash = Action.entry_hash
+                    WHERE DhtOp.when_integrated IS NOT NULL
+                    AND DhtOp.validation_status = :status
+                    ORDER BY DhtOp.when_integrated ASC
```

Comment:
```suggestion
                    SELECT
                        DhtOp.hash as hash, DhtOp.type as dht_type, DhtOp.validation_status,
                        Action.blob as action_blob, Entry.blob as entry_blob
                    FROM DhtOp
                        JOIN Action 
                            ON Action.hash = DhtOp.action_hash
                        LEFT JOIN Entry 
                            ON Entry.hash = Action.entry_hash
                    WHERE 
                        DhtOp.when_integrated IS NOT NULL
                        AND DhtOp.validation_status = :status
                    ORDER BY
                        DhtOp.when_integrated ASC
```

Some indentation would be nice please :)

Reply:
https://github.com/holochain/holochain/pull/5329/commits/79361a265b123fc04c68986fabe8420b32fccfa2